### ISSUE TITLE
No Bitcoin on Kraken ?

### ISSUE THREAD

**Issue by Foul** [MarcinOrlowski/crypto-tracker-plasmoid#34]
Can't find Bitcoin listed when choosing Kraken ????

**Comment by MarcinOrlowski** [comment]
Indeed. I'd say that most likely there have been a reason for that. But I cannot remember much, so it can also be a simple bug. Will check.

**Comment by Razer0123** [comment]
Any news?
